.github/workflows/build-and-test.yml

```diff
@@ -10,6 +10,8 @@ on:
 
 env:
   GRADLE_OPTS: -Dorg.gradle.daemon=false -Dkotest.assertions.multi-line-diff=unified
+  REGISTRY: ghcr.io
+  TEST_IMAGE_TAG: ort:test
```

Comment:
Just a though: Should we name this `FUNTEST_...` and `ort:funtest` to make more clear it's only used for functional tests?

Reply:
I'm fine with both, if you want I can change it.

Reply:
Let's keep it as-is then to reduce the iterations for this PR. Maybe we'll use the image also for integration or performance tests, and I can't make up my mind whether it should rather be `FUN_TEST_...` and `ort:funTest` instead 😅